tests/foreman/ui/test_myaccount.py

```diff
@@ -15,24 +15,90 @@
 
 @Upstream: No
 """
+import random
 
+from fauxfactory import gen_string, gen_alpha
+from nailgun.entities import User
+from robottelo.constants import LANG_TO_LOCALE_DCT
 from robottelo.decorators import stubbed, tier1
 from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators
+from robottelo.ui.locators import locators
+from robottelo.ui.my_account import MyAccount
+from robottelo.ui.session import Session
+
+
+def _valid_string_data(min_len=1, max_len=50):
+    """Generates valid string data for ui tests
+
+    :param min_len: min len for generated string
+    :param max_len: max len for generated string. Default is 50 because of
+    ui validation
+    :return: list of generated strings
+    """
+    return [
+        gen_string('alphanumeric', random.randint(min_len, max_len)),
+        gen_string('alpha', random.randint(min_len, max_len)),
+        gen_string('cjk', random.randint(min_len, max_len)),
+        gen_string('latin1', random.randint(min_len, max_len)),
+        gen_string('numeric', random.randint(min_len, max_len)),
+        gen_string('utf8', random.randint(min_len, max_len)),
+    ]
 
 
 class MyAccountTestCase(UITestCase):
     """Implements Users tests in UI
 
     [1] Positive Name variations - Alpha, Numeric, Alphanumeric, Symbols,
-    Latin1, Multibyte, Max length,  Min length, Max_db_size, html, css,
+    Latin1, Multibyte, Max length,  Min length, Max_db_size, css,
     javascript, url, shell commands, sql, spaces in name
 
-    [2] Negative Name Variations -  Blank, Greater than Max Length,
+    [2] Negative Name Variations -  html, Blank, Greater than Max Length,
     Lesser than Min Length, Greater than Max DB size
 
     """
 
-    @stubbed()
+    def setUp(self):
+        """Setup of myaccount"""
+        super(MyAccountTestCase, self).setUp()
+        self.my_account = MyAccount(self.browser)
+        user = User(login=gen_alpha(), password='password')
+        user.id = user.create().id
```

Comment:
maybe you can add a comment here `# reassign the created user id because password should be persisted in object` or other explanation on why you did not uses the common `self.user = User().create()`